t/helper/auth.t

```diff
@@ -0,0 +1,206 @@
+use Mojo::Base -strict;
+
+use Test::More;
+use Test::Mojo;
+
+use Bytes::Random::Secure qw(random_string_from);
+use Crypt::Bcrypt         qw(bcrypt bcrypt_check);
+use Crypt::URandom        qw(urandom);
+
+use CtrlNest::Helper::Auth;
+use CtrlNest::Helper::Constants;
+
+use lib 't';
+use util::common;
+use util::init;
+
+# Init Mojo & Schema
+my ($t, $db) = init_tests();
+
+################################################################################
+
+subtest 'Test helper method validate_auth() with random data' => sub {
+  for (1 .. 4) {
+
+    # Create a random valid username
+    my $username = generate_random_username(
+      $_ % 2 == 0 ? USERNAME_MIN_LEN : USERNAME_MAX_LEN);
+
+    # Create a random valid password
+    my $password = generate_random_password(
+      $_ % 2 == 0 ? PASSWORD_MIN_LEN : PASSWORD_MAX_LEN);
+
+    # Create a random user
+    my $user = create_user($db, $username, $password);
+    ok(defined $user);
+
+    # The password must mach
+    $user = validate_auth($t->app, $username, $password);
+
+    # The user must be returned
+    ok(defined $user);
+    ok(defined $user->{username});
+    ok(defined $user->{password});
+    ok(defined $user->{role});
+
+    # Verify username
+    ok($user->{username} eq $username);
+
+    # The password most not pach if at least one char si different
+    $user = validate_auth($t->app, $username, $password . 'X');
+    is($user, undef);
+
+    $user = validate_auth($t->app, $username . 'X', $password);
+    is($user, undef);
+  }
+};
+
+################################################################################
+
+# subtest 'Test helper method validate_credentials() with random data' => sub {
```

Comment:
You are right, we don't need this test because we that that function with random data already, but get rid of the comments